app/components/FilterOptions.tsx

```diff
@@ -37,6 +39,8 @@ const FilterOptions = ({
 
   const selectedLabel = selected ? `${selectedPrefix} ${selected.label}` : "";
 
+  const isAuthorDropdown = defaultLabel === "Any author" ? true : false;
```

Comment:
Doesn't seem to account for translations. Could you test this for language other than en?

Reply:
Oops! Missed that scenario. fixing it